src/NServiceBus.SagaAudit/ServiceControlBackend.cs

```diff
@@ -38,8 +37,9 @@ async Task Send(object messageToSend, TimeSpan timeToBeReceived, TransportTransa
             try
             {
                 var outgoingMessage = new OutgoingMessage(Guid.NewGuid().ToString(), headers, body);
-                var operation = new TransportOperation(outgoingMessage, new UnicastAddressTag(destinationQueue), deliveryConstraints: new List<DeliveryConstraint> { new DiscardIfNotReceivedBefore(timeToBeReceived) });
-                await messageSender.Dispatch(new TransportOperations(operation), transportTransaction, new ContextBag()).ConfigureAwait(false);
+
+                var operation = new TransportOperation(outgoingMessage, new UnicastAddressTag(destinationQueue), new DispatchProperties { DiscardIfNotReceivedBefore = new DiscardIfNotReceivedBefore(timeToBeReceived) });
```

Comment:
Maybe not worth it but I recall the destinationQueue is always the same so theoretically `UnicastAddressTag` could be reused from a field

Reply:
I don't know anything about that, so I'd prefer not to make any changes to non-token related stuff.